### DISCUSSION TITLE
RP Model B Rev 2 update error

### DISCUSSION THREAD

**Discussion by surfninja2** [rern/rAudio#668] [Fixed Issues]
I'm getting errors when updating my rAudio.  Is this a problem?  Or a feature?

/srv/http/settings/relays.php
/srv/http/settings/system.php
/srv/http/settings/system-i2s.json
**/srv/http/bash/cmd.sh: line 1043: echo: write error: Broken pipe**

I have rAudio on several RPs (RP2 model B, Zero, old Model B, rev 2) and this is the only one that fails.

This is the first problem I noticed with rAudio.  What a great music player system. Great job Rern.  I tell everyone this is the music server they should use.

**Comment by rern** [comment]
Which page or where did the error show?
Let me see: `aplay -l`

**Reply by surfninja2** [reply]
sorry for the delay  in the response.

ADDONS-> Update Button
the screen shows the get and files being updated and dies after the system-i2s.json get

here is a screen shot i moved to google drive
https://drive.google.com/file/d/1AcqknUX3OySDa5lOIMI-OQMl_9zufR8R/view?usp=sharing

the RP seems to work using rAudio but don't know if the update finished or it is running the older version.

**Comment by surfninja2** [comment]
rAudio-1:~  #  aplay -l
**** List of PLAYBACK Hardware Devices ****
card 0: RPiCirrus [RPi-Cirrus], device 0: WM5102 AiFi wm5102-aif1-0 [WM5102 AiFi wm5102-aif1-0]
  Subdevices: 0/1
  Subdevice #0: subdevice #0

**Comment by surfninja2** [comment]
I think this can be closed.  I just upgraded my old model B to 20220107 without any errors.
